src/FffGcodeWriter.h

```diff
@@ -498,9 +508,15 @@ class FffGcodeWriter : public SettingsMessenger, NoCopy
      * \param mesh_config the line config with which to print a print feature
      * \param skin_part The skin part for which to create gcode
      * \param layer_nr The current layer number.
+     * \param skin_overlap The distance by which the skin overlaps with the wall insets and the distance by which the perimeter gaps overlap with adjacent print features.
+     * \param skin_angle The angle in the XY plane at which the infill is generated.
+     * \param pattern The pattern with which we'd like to fill skin areas (unless we bridge, then we force the lines pattern)
+     * \param z The (approximate) height at which we are adding gcode plans
```

Comment:
We are adding to the layer plans. How would you call it? What are we adding to the layer plans if not 'gcode plans'?